### ISSUE TITLE
instructions

### ISSUE THREAD

**Issue by Virtis83x** [DBuit/sidebar-card#94]
I am very new to this and the instructions don't make any sense.

Can this be dumbed down for me

**Comment by denisfrench** [comment]
I agree that the instructions are confusing partly because they assume the the UI is in yaml mode (not the default), and also because when in yaml mode `resources:` has been moved to `configuration.yaml`, and `views:` is now in separate yaml files for individual dashboards.

It is therefore unclear whether the configuration for sidebar-card should be entered into `configuration.yaml`, or one (or all) of the separate dashboard yaml files.

Please consider updating the configuration instructions so they are consistant with Home Assistant documentation at https://www.home-assistant.io/dashboards/dashboards/.

**Comment by DBuit** [comment]
I will look into this for my own configuration it will use one yaml as in the old days :)

**Comment by fransboumans** [comment]
For anyone (in the future) who is still looking for up-to-date instruction:

1. Install `sidebar-card` through HACS or manual ([original instructions](https://github.com/DBuit/sidebar-card?tab=readme-ov-file#installation-instructions)). Don't forget to download `side-card` through HACS once you added the repository URL! (if installing through HACS)
2. Open the dashboard you want to enable `sidebar-card` and click on the three dots ( **⋮** ) in the upper right corner and select **"Edit dashboard"**
3. Click once more on the three dots ( **⋮** ) in the upper right corner and select **"Manage resources"** and click on **"Add resources"** (blue button right bottom corner) and enter the following the URL field depending on which install method you used:

    **HACS:** /hacsfiles/sidebar-card/sidebar-card.js
    **Manual:** /local/sidebar-card.js
    
    The 'JavaScript module' radio button under 'Resource type' should enable automatically. If not, do so. Click on **"CREATE"**. 

4. Navigate back to the dashboard and click on the three dots ( **⋮** ) in the upper right corner and select **"Raw configuration editor"** and enter the following:
```
sidebar:
  # Displays a title at the top of the side-bar, this is optional and can be removed if not needed
  title: Sidebar title
  # Hides the Home Assistant default sidebar
  hideHassSidebar: true
  # Hides the Home Assistant default top menu
  hideTopMenu: true
views:
  ...
```

5. Depending on what you want, you can enable or disable the Home Assistant default sidebar by setting `hideHassSidebar` and the Home Assistant menu at the top by setting `hideTopMenu`. Other options can be found [here](https://github.com/DBuit/sidebar-card?tab=readme-ov-file#main-options). I noticed that a hard refresh is sometimes needed to trigger the changes (Ctrl - F5).

**Tip:** when you have hidden the sidebar and/or top menu but need to access them, you can add `?sidebarOff` at the back of the URL in your browser.

I hope this helps! 😃

**Comment by mikecarson89** [comment]
I am having the same issue as Null-Mind. Did you managed to solve it?

**Comment by stevland** [comment]
> For anyone (in the future) who is still looking for up-to-date instructions:
> 
> 1. Install `sidebar-card` through HACS or manual ([original instructions](https://github.com/DBuit/sidebar-card?tab=readme-ov-file#installation-instructions)). Don't forget to download `side-card` through HACS once you added the repository URL! (if installing through HACS)
> 2. Open the dashboard you want to enable `sidebar-card` and click on the three dots ( **⋮** ) in the upper right corner and select **"Edit dashboard"**
> 3. Click once more on the three dots ( **⋮** ) in the upper right corner and select **"Manage resources"** and click on **"Add resources"** (blue button right bottom corner) and enter the following the URL field depending on which install method you used:
>    **HACS:** /hacsfiles/sidebar-card/sidebar-card.js
>    **Manual:** /local/sidebar-card.js
>    The 'JavaScript module' radio button under 'Resource type' should enable automatically. If not, do so. Click on **"CREATE"**.
> 4. Navigate back to the dashboard and click on the three dots ( **⋮** ) in the upper right corner and select **"Raw configuration editor"** and enter the following:
> 
> ```
> sidebar:
>   # Displays a title at the top of the side-bar, this is optional and can be removed if not needed
>   title: Sidebar title
>   # Hides the Home Assistant default sidebar
>   hideHassSidebar: true
>   # Hides the Home Assistant default top menu
>   hideTopMenu: true
> views:
>   ...
> ```
> 
> 5. Depending on what you want, you can enable or disable the Home Assistant default sidebar by setting `hideHassSidebar` and the Home Assistant menu at the top by setting `hideTopMenu`. Other options can be found [here](https://github.com/DBuit/sidebar-card?tab=readme-ov-file#main-options). I noticed that a hard refresh is sometimes needed to trigger the changes (Ctrl - F5).
> 
> **Tip:** when you have hidden the sidebar and/or top menu but need to access them, you can add `?sidebarOff` at the back of the URL in your browser.
> 
> I hope this helps! 😃

I can't imagine how much time you saved me with these instructions. Ty!
